tests/suites/smoke/deploy.sh

```diff
@@ -34,9 +34,27 @@ run_charmhub_deploy() {
 
 	ensure "${model_name}" "${file}"
 
+	# Deploy juju-qa-test from charmhub.
 	charm="juju-qa-test"
 	juju deploy "$charm" --revision 22 --channel stable "$charm"
-	wait_for "$charm" "$(idle_condition "$charm")"
+	# Also deploy juju-qa-dummy-source and juju-qa-dummy-sink from charmhub
+	# to test other code paths. 
+	# TODO(nvinuesa): Ideally, we want to deploy the next 2 charms with a 
+	# placement directive, both to make the test faster and also to test 
+	# the placement directives. However, until machines are not fully 
+	# migrated to dqlite, placement directives are broken. 
+	charmSource="juju-qa-dummy-source"
```

Comment:
Bit weird to declare a variable just to use it on the next line and never again

Reply:
We also use snake_case not camelCase in shell tests.

Reply:
Yeah sorry, IDK why I did the same as it was previously done in the test. I removed this variables now.
@hpidcock too much Go and no play makes Nico a dull boy.